src/nvim/ex_cmds.c

```diff
@@ -4312,13 +4332,7 @@ void global_exe(char_u *cmd)
   old_lcount = curbuf->b_ml.ml_line_count;
 
   while (!got_int && (lnum = ml_firstmarked()) != 0 && global_busy == 1) {
-    curwin->w_cursor.lnum = lnum;
-    curwin->w_cursor.col = 0;
-    if (*cmd == NUL || *cmd == '\n') {
-      do_cmdline((char_u *)"p", NULL, NULL, DOCMD_NOWAIT);
-    } else {
-      do_cmdline(cmd, NULL, NULL, DOCMD_NOWAIT);
-    }
+    global_exe_one(cmd, lnum);
```

Comment:
You left out the `*_global_changes()` around this line. They got renamed to `*_batch_changes()` in Nvim.

Reply:
@justinmk removed it in cc7e344f8357d07b1df17df0b322152d5c50739b

Reply:
D'oh! I just grep'd it in master and saw a few occurrences. Well done. :]